Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockGUI.php

```diff
@@ -523,6 +540,12 @@ public function manageObject()
                 $group->setItems([]);
                 foreach ($items as $item) {
```

Comment:
I would suggest using `array_filter` instead of a foreach, as you are just filtering an array.